src/index.html

```diff
@@ -115,7 +129,7 @@ <h3 class="product__params-title">{{{ label }}}</h3>
                 <div class="widget-amount"> <!-- widget-amount -->
                   <h3 class="product__params-title">Select quantity</h3>
                   <a class="btn-quantity" href="#less"><i class="fas fa-minus"></i></a>
-                  <input type="text" name="amount" value="1">
+                  <input type="text" name="amount">
```

Comment:
Jeśli usuniesz atrybut value i nie ustawisz domyślnej wartości w constructorze `AmountWidget`, to początkowa wartość wyniesie `undefined`

Reply:
thisWidget.getElements(element);
      thisWidget.setValue(settings.amountWidget.defaultValue);
      thisWidget.initActions();
    }

Reply:
Wszystko się zgadza, ale w kodzie masz:
```js
thisWidget.getElements(element);
thisWidget.setValue(settings.amountWidget.defaultValue);
thisWidget.initActions();
```
zobacz linijkę 233 w `script.js`. Chyba, że zmiany masz w innym branchu?

Reply:
https://github.com/MartynaCylke/project-pizzeria/pull/24/commits/3c37662e30ca9fc74a0bc652b39ead69b7177cca

265 na zielono,
tak te branche mam cos powalone